examples/quantizing_moe_fp8/mixtral_moe.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+from transformers import AutoTokenizer
+
```

Comment:
Could we add an int4 example? vLLM at the moment only supports int4 models. 

It would be good to add that example and then validate that it runs smoothly by adding it to our e2e tests on vLLM.

Reply:
Will add that as a follow up task, could you create a ticket to track?